src/Uno.UI.FluentTheme.v2/Controls/FauxGradientBottomBorder.cs

```diff
@@ -0,0 +1,84 @@
+﻿#nullable enable
+
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace Uno.UI.FluentTheme.Controls
+{
+	internal class FauxGradientBottomBorder : ContentControl
+	{
+#if __WASM__ || __IOS__ || __ANDROID__
+		private readonly Border? _displayBorder = null;
+#endif
+
+		public FauxGradientBottomBorder()
+		{
+#if __WASM__ || __IOS__ || __ANDROID__
+			HorizontalContentAlignment = Windows.UI.Xaml.HorizontalAlignment.Stretch;
+			VerticalContentAlignment = Windows.UI.Xaml.VerticalAlignment.Stretch;
+			Content = _displayBorder = new Border();
+#endif
+		}
+
+		public Brush RequestedBorderBrush
+		{
+			get => (Brush)GetValue(RequestedBorderBrushProperty);
+			set => SetValue(RequestedBorderBrushProperty, value);
+		}
+
+		public static DependencyProperty RequestedBorderBrushProperty { get; } =
+			DependencyProperty.Register(
+				nameof(RequestedBorderBrush),
+				typeof(Brush),
+				typeof(FauxGradientBottomBorder),
+				new PropertyMetadata(null, propertyChangedCallback: (s, args) => (s as FauxGradientBottomBorder)?.OnBorderChanged()));
```

Comment:
Should this be `FrameworkPropertyMetadata`? If yes, we will need to add a `Directory.Build.targets` to FluentTheme directories similar to this:

https://github.com/unoplatform/uno/blob/master/src/Uno.UI/Directory.Build.targets

Reply:
First time I noticed this discussion. That could open up a whole can of worms on my end since `PropertyMetadata` is always used. Additionally, `FrameworkPropertyMetadata` does NOT exist in UWP. So what is going on here?

I will comment in the original issue I guess.

Reply:
It should be, as a general rule, though it won't change much in this case.